assets/helpers/externalLinks.js

```diff
@@ -216,14 +214,45 @@ function getDigitalCheckout(
 }
 
 
+function convertCountryGroupIdToAppStoreCountryCode(cgId: CountryGroupId) {
+  const groupFromId = countryGroups[cgId];
+  if (groupFromId) {
+    switch (groupFromId.supportInternationalisationId.toLowerCase()) {
+      case 'uk':
+        return 'gb';
+      case 'int':
+        return 'us';
+      case 'eu':
```

Comment:
There isn't an app store with euros?

Reply:
No, they support individual country codes, but not a EU according to their list here: https://affiliate.itunes.apple.com/resources/documentation/linking-to-the-itunes-music-store/#CountryCodes

👍for tests